frigate/http.py

```diff
@@ -1647,11 +1648,11 @@ def latest_frame(camera_name):
             + retry_interval
         ):
             if current_app.camera_error_image is None:
-                error_image = glob.glob("/opt/frigate/frigate/images/camera-error.jpg")
+                error_image = INSTALL_DIR / "frigate/images/camera-error.jpg"
 
-                if len(error_image) > 0:
+                if error_image.is_file():
                     current_app.camera_error_image = cv2.imread(
-                        error_image[0], cv2.IMREAD_UNCHANGED
+                        error_image, cv2.IMREAD_UNCHANGED
```

Comment:
This doesn't work in my testing `TypeError: Can't convert object to 'str' for 'filename'`

Reply:
Yeah, will need a type cast. Also taking a look at the typing checks.